src/base/memory_monitor_sr.cpp

```diff
@@ -51,3 +95,20 @@ namespace cubmem
     return filecopy + ':' + std::to_string (line);
   }
 }
+
+using namespace cubmem;
+
+bool mmon_is_mem_tracked ()
```

Comment:
NIT:

```suggestion
bool mmon_is_tracked_mem ()
```

Reply:
This function check whether memory monitoring option is on or not. But I think your suggestion could get meaning like "This memory is tracked (about the memory pointer)" rather than "cubrid memory monitoring option is turned on". What about `mmon_is_mem_traced()`?

Reply:
If you just want to show the meaning `cubrid memory monitoring option is turned on`, then how about `mmon_is_memory_monitoring_enabled`?

Reply:
I change it to `mmon_is_memory_monitor_enabled`.